sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -3,10 +3,10 @@
 import logging
 from typing import Sequence, Any
 
+from opentelemetry._logs.severity import SeverityNumber
 from opentelemetry.semconv.trace import SpanAttributes
-from opentelemetry.sdk._logs import LogData
-from opentelemetry.sdk._logs.severity import SeverityNumber
-from opentelemetry.sdk._logs.export import LogExporter, LogExportResult
+from opentelemetry.sdk._logs._internal import LogData
+from opentelemetry.sdk._logs._internal.export import LogExporter, LogExportResult
```

Comment:
These should be fine as is and don't need the `_internal` added since it's made available [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py).

Reply:
Pyright yelled at me to make that change.

Reply:
Interesting. Not sure why pyright would be complaining. Was this in your IDE or was the CI complaining? 
These imports seem to be resolving okay as is on my end with no complants.

I think for consistency with other places where these are imported (e.g. in opentelemetry-python code or in the `test_logs.py` file here), we should remove the `_internal` parts of the import paths.

Reply:
<img width="674" alt="image" src="https://user-images.githubusercontent.com/301208/207439964-ec15163a-1cb9-436d-96d0-977b31530cdd.png">

Reply:
Pushed an updated change